docs/usage.md

```diff
@@ -68,10 +69,26 @@ An [example samplesheet](../assets/samplesheet.csv) has been provided with the p
 
 > ⚠️ We highly recommend performing quality control on input contigs before running the pipeline. You may not receive results for some tools if none of the contigs in a FASTA file reach certain thresholds. Check parameter documentation for relevant minimum contig parameters.
 
-## Notes on screening tools
+## Notes on screening tools and taxonomic classification
 
 The implementation of some tools in the pipeline may have some particular behaviours that you should be aware of before you run the pipeline.
 
+### MMseqs2
+
+MMseqs2 is currently the only taxonomic classification tool used in the pipeline to assign a taxonomic lineage to the input contigs. The database used to assign the taxonomic lineage can either be:
+
+- a custom based database created by the user using `mmseqs createdb` externally and beforehand. If this flag is assigned, this database takes precedence over the default database in ` mmseqs_databases_id`.
+
+```
+taxa_classification_mmseqs_databases_localpath 'path/to/mmsesqs_custom_database/dir'
+```
+
+- an MMseqs2 ready database. These databases were compiled by the developers of MMseqs2 and can be called using their labels. All available options can be found [here](https://github.com/soedinglab/MMseqs2/wiki#downloading-databases). Only use those databases that have taxonomy files available (i.e., Taxonomy == Yes). By default mmseqs2 in the pipeline uses 'Kalamari' and runs an aminoacid based alignment.
```

Comment:
Maybe provide a link to a description of what is in 'kalamari'? if there is one...

```suggestion
- an MMseqs2 ready database. These databases were compiled by the developers of MMseqs2 and can be called using their labels. All available options can be found [here](https://github.com/soedinglab/MMseqs2/wiki#downloading-databases). Only use those databases that have taxonomy files available (i.e., Taxonomy == Yes). By default mmseqs2 in the pipeline uses '[Kalamari](URL TO GO HERE)' and runs an aminoacid based alignment.
```